pkg/userdata/ubuntu/userdata.go

```diff
@@ -133,6 +133,9 @@ hostname: {{ .MachineSpec.Name }}
 # Never set the hostname on AWS nodes. Kubernetes(kube-proxy) requires the hostname to be the private dns name
 {{ end }}
 
+groups:
```

Comment:
What for do we need this?

Reply:
I would like to stick as close as possible to "normal" setup of docker as possible.
Using a `docker` group is the common way of handling docker permission, no?

Reply:
Oh wait, i just realized that we don't actually use that group anywhere

Reply:
Removed with https://github.com/kubermatic/machine-controller/pull/382/commits/b4f71d6db2a40962ef43b1169d630ae4c4e6ea6f

Reply:
We did use it on the socket ;)

Reply:
Oh that explains a lot :D